docs/conf.py

```diff
@@ -0,0 +1,26 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "Moonraker Home Assistant"
+copyright = "2023, Marc-Olivier Arsenault, Eric Tremblay"
+author = "Marc-Olivier Arsenault, Eric Tremblay"
```

Comment:
Is it possible to add a field for "contributor" for those who submit patches but are not actively part of the project